chef/cookbooks/horizon/templates/suse/openstack-dashboard.conf.erb

```diff
@@ -46,6 +46,9 @@
     DocumentRoot <%= @horizon_dir %>
     Alias /media <%= @horizon_dir %>/media
     Alias /static <%= @horizon_dir %>/static
+    <% if @grafana_available %>
```

Comment:
I would do it the same way that is already done at line 33:
`<% unless @ssl_crt_chain_file.nil? or @ssl_crt_chain_file.empty? %>`